providers/ruleset.rb

```diff
@@ -18,9 +18,12 @@
 #
 
 # provider for installing audit rules from a template
+
 action :create do
-  template '/etc/audit/audit.rules' do
+  t = template '/etc/audit/audit.rules' do
     source "#{new_resource.name}.erb"
-    notifies :restart, resources(service: 'auditd')
+    notifies :run, 'execute[auditctl -R]', :immediately
```

Comment:
I am working on a PR to bring this cookbook into compliance with the 2.3 changes (moving from a single file to multiple rule files) and the USE_AUGENRULES flag is needed on debian systems to turn the rules.d usage on.  This change will just get reverted in my PR :p